input/fsh/Instance-Specimen.fsh

```diff
@@ -1,22 +1,38 @@
-// All instances of type Specimen should go in this file.
+/** 
+ All instances of Specimen profile should go in this file.
+ */
 
-Alias: SpecimenTypeCodeSystem = http://terminology.hl7.org/CodeSystem/v2-0487
-Alias: SCT = http://snomed.info/sct
-
-Instance: bs-001
+Instance: specimen-example-1
 InstanceOf: Specimen
-* meta.profile = "http://hl7.org/fhir/StructureDefinition/Specimen"
-* identifier[0].system = "https://kf-api-dataservice.kidsfirstdrc.org/biospecimens?study_id=SD_BHJXBDQK"
-* identifier[0].value = "571312"
-* identifier[1].system = "urn:kids-first:unique-string"
-* identifier[1].value = "Specimen|SD_BHJXBDQK|571312"
-* type.coding = SpecimenTypeCodeSystem#TISS "Tissue"
+Usage: #example
+Description: "Example of the representation of Specimen."
+* type = $v2-0487#TISS "Tissue"
 * type.text = "Solid Tissue"
 * subject = Reference(patient-example-1)
-* receivedTime = "2019-06-15"
-* collection.collectedDateTime = "2019-06-15"
+* receivedTime.extension.url = $cqf-relativeDateTime
```

Comment:
Good call on the alias here. I hadn't even thought about it appearing multiple times.